src/probestack.rs

```diff
@@ -41,88 +41,4 @@
 //! probes on any other architecture like ARM or PowerPC64. LLVM I'm sure would
 //! be more than welcome to accept such a change!
 
-#![cfg(not(windows))] // Windows already has builtins to do this
-
-#[naked]
-#[no_mangle]
-#[cfg(all(target_arch = "x86_64", not(feature = "mangled-names")))]
-pub unsafe extern "C" fn __rust_probestack() {
```

Comment:
I don't think we'll want to delete these because a C compiler isn't always available and we'll still want definitions of these functions.

Reply:
Actually, we wouldn't really need C because we are compiling assembly files. Passing '.S' files to a C compiler is just wrapping around an assembler. Can we rely solely on our bundled-in LLVM for it somehow?

Reply:
Nah unfortunately we don't ship an assembler that uses LLVM, so we're left basically looking for `cc`. You're right that we otherwise just need `as`, but that's not actually that much easier to acquire than a C compiler most of the time.